codecs/zstd/build.sh

```diff
@@ -49,7 +49,7 @@ cd ../../../
     ${OPTIMIZE} \
     -I "$CODEC_DIR/lib" \
     --closure 1 \
-    -fwasm-exceptions
+    -fwasm-exceptions \
```

Comment:
Ah, I see you did use wasm exceptions!

Reply:
I believe this is listed as "legacy" now, but the alternative proposal isn't supported in any runtimme at all so I'm good with using for now: https://webassembly.org/features/